pontoon/base/views.py

```diff
@@ -103,30 +103,51 @@ def locale_projects(request, locale):
 def locale_stats(request, locale):
     """Get locale stats used in All Resources part."""
     locale = get_object_or_404(Locale, code=locale)
-    return JsonResponse(locale.stats(), safe=False)
+    stats = TranslatedResource.objects.filter(locale=locale).string_stats(request.user)
+    stats["title"] = "all-resources"
+    return JsonResponse([stats], safe=False)
 
 
 @utils.require_AJAX
 def locale_project_parts(request, locale, slug):
     """Get locale-project pages/paths with stats."""
-    try:
-        locale = Locale.objects.get(code=locale)
-    except Locale.DoesNotExist as e:
-        return JsonResponse(
-            {"status": False, "message": f"Not Found: {e}"},
-            status=404,
-        )
 
     try:
+        locale = Locale.objects.get(code=locale)
         project = Project.objects.visible_for(request.user).get(slug=slug)
-    except Project.DoesNotExist as e:
+        tr = TranslatedResource.objects.filter(
+            resource__project=project, resource__entities__obsolete=False, locale=locale
+        ).distinct()
+        details = list(
+            tr.annotate(
+                title=F("resource__path"),
+                total=F("total_strings"),
+                pretranslated=F("pretranslated_strings"),
+                errors=F("strings_with_errors"),
+                warnings=F("strings_with_warnings"),
+                unreviewed=F("unreviewed_strings"),
+                approved=F("approved_strings"),
+            )
+            .order_by("title")
+            .values(
+                "title",
+                "total",
+                "pretranslated",
+                "errors",
+                "warnings",
+                "unreviewed",
+                "approved",
+            )
+        )
+        all_res_stats = tr.string_stats(request.user, show_disabled=True)
```

Comment:
Why? Same question for other places where we use `show_disabled=True`.

Reply:
I think I know why. It's related to my question above? It's to show the right data for system projects, right?